packages-ts/starknet-gauntlet-emergency-protocol/src/commands/starknetValidator/addAccess.ts

```diff
@@ -0,0 +1,38 @@
+import { EVMExecuteCommandConfig, makeEVMExecuteCommand } from '@chainlink/evm-gauntlet'
+import { CONTRACT_LIST, starknetValidatorContractLoader } from '../../lib/contracts'
+import { CATEGORIES } from '../../lib/categories'
+
+export interface UserInput {
+  user: string
+}
+
+type ContractInput = [_user: string]
+
+const makeContractInput = async (input: UserInput): Promise<ContractInput> => {
+  return [input.user]
+}
+
+const makeUserInput = async (flags): Promise<UserInput> => {
+  if (flags.input) return flags.input as UserInput
+  return {
+    user: flags.user,
+  }
+}
+
+const commandConfig: EVMExecuteCommandConfig<UserInput, ContractInput> = {
+  contractId: CONTRACT_LIST.STARKNET_VALIDATOR,
+  category: CATEGORIES.STARKNET_VALIDATOR,
+  action: 'addAccess',
```

Comment:
Let's use snake_case on the action to be consistent with rest of commands. As the contract function is camelCase, we can specify it in `internalFunction`
```suggestion
  internalFunction: 'addAccess',
  action: 'add_access',
```

Reply:
This applies to every command